TESTS/mbedmicro-rtos-mbed/basic/main.cpp

```diff
@@ -19,7 +19,7 @@
 #include "utest/utest.h"
 #include "unity/unity.h"
 
-#if defined(SKIP_TIME_DRIFT_TESTS) || defined(MBED_RTOS_SINGLE_THREAD) || !DEVICE_USTICKER
+#if defined(SKIP_TIME_DRIFT_TESTS) || defined(MBED_RTOS_SINGLE_THREAD) || !DEVICE_USTICKER || !defined(MBED_CONF_RTOS_PRESENT)
 #error [NOT_SUPPORTED] test not supported
```

Comment:
please breakdown the qualifier macro checks:
```cpp
#if defined(SKIP_TIME_DRIFT_TESTS) || 
#error [NOT_SUPPORTED] Time drifting tests skipped
#elif defined(MBED_RTOS_SINGLE_THREAD) || !defined(MBED_CONF_RTOS_PRESENT)
#error [NOT_SUPPORTED] Test requires RTOS and threads to run
#elif !DEVICE_USTICKER
#error [NOT_SUPPORTED] USTicker is need to be enabled to run this test
```

Reply:
Discussed with Jamesbeyond and split into two ifdef